themes-default/slim/views/schedule.mako

```diff
@@ -19,14 +21,34 @@ const startVue = () => {
                 header: 'Schedule'
             };
         },
+        computed: Object.assign(
+            puex.mapState(['config', 'shows']),
+            {
+                layout: {
+                    get() {
+                        const { config } = this;
+                        return config.layout.schedule;
+                    },
+                    set(layout) {
+                        const { $store } = this;
+                        const page = 'schedule';
+                        $store.dispatch('setLayout', { page, layout });
```

Comment:
Please only use this for updating the layout. Just provide it with the page and layout you want to set. It'll `PATCH` to the API and then update the config in the store.